tests/unit/cirrus/harnesses.py

```diff
@@ -11,11 +11,19 @@
 import unittest
 import tempfile
 import mock
+import subprocess
 import ConfigParser
 
 from cirrus.configuration import Configuration
 
 
+def _repo_directory():
```

Comment:
I would remove the private-implying `_` prefix if this is going to be imported and used by other modules.